nav2_bt_navigator/behavior_trees/navigate_w_replanning_and_recovery.xml

```diff
@@ -7,22 +7,22 @@
     <RecoveryNode number_of_retries="6" name="NavigateRecovery">
       <Sequence name="NavigateWithReplanning">
         <RateController hz="1.0">
-          <Fallback name="GoalReached">
+          <Fallback name="GoalReachedFallback">
             <GoalReached/>
-            <RecoveryNode number_of_retries="1" name="ComputePath">
+            <RecoveryNode number_of_retries="1" name="ComputePathRecovery">
               <ComputePathToPose goal="{goal}" path="{path}"/>
-              <ClearEntireCostmap service_name="global_costmap/clear_entirely_global_costmap"/>
+              <ClearEntireCostmap  name="ClearLocalCostmap-CP" service_name="global_costmap/clear_entirely_global_costmap"/>
             </RecoveryNode>
           </Fallback>
         </RateController>
-        <RecoveryNode number_of_retries="1" name="FollowPath">
+        <RecoveryNode number_of_retries="1" name="FollowPathRecovery">
           <FollowPath path="{path}"/>
-          <ClearEntireCostmap service_name="local_costmap/clear_entirely_local_costmap"/>
+          <ClearEntireCostmap name="ClearLocalCostmap-FP" service_name="local_costmap/clear_entirely_local_costmap"/>
         </RecoveryNode>
       </Sequence>
       <SequenceStar name="RecoveryActions">
-        <ClearEntireCostmap service_name="local_costmap/clear_entirely_local_costmap"/>
-        <ClearEntireCostmap service_name="global_costmap/clear_entirely_global_costmap"/>
+        <ClearEntireCostmap name="ClearLocalCostmap-Top" service_name="local_costmap/clear_entirely_local_costmap"/>
```

Comment:
Its not clear to me why this "name" is here or how it relates to the logging description of the PR. I'm certainly guilt of adding in multiple things in a PR but can you explain this one for me?

Also what's `TOP` `FP` and `CP`?

Reply:
I wanted every node in the tree to have a unique name, otherwise you can't easily tell which status change goes with which node. Without a distinct name, each ClearEntireCostmap would be indistinguishable.

The `Top`, `FP` and `CP` were supposed to be shorthand for `Top level`, `FollowPath`, and `ComputePath`. In other words, they were indicating which part of the tree they were in. I'll grant you the names suck. I was trying to stick to a 25 character limit, which is what was originally used in the Behavior Tree library `cout` logger. Anything longer would mess up the formatting.

Reply:
Silly Q, 

Then why not just the clear local/global costmap? I think theres some implicit understanding a local costmap relates to a controller and global to a planner.